src/TesApi.Web/BatchScheduler.cs

```diff
@@ -447,7 +447,8 @@ private static string StandardizeStartTaskPath(string startTaskPath, string defa
         /// <param name="inputFile"><see cref="Tes.Models.TesInput"/> file</param>
         /// <returns>True if the file is a Cromwell command script</returns>
         private static bool IsCromwellCommandScript(TesInput inputFile)
-            => inputFile.Name.Equals("commandScript");
+            // See https://github.com/broadinstitute/cromwell/blob/17efd599d541a096dc5704991daeaefdd794fefd/supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala#L58
```

Comment:
nit: is having a hard link to the line number sustainable to have in the codebase? Could we link to the file and say look at `private val commandScript`?

Reply:
For me it's easier to move through the file history easier this way, to see if it's still relevant, rather than searching for something that may have been renamed between the reference and the subsequent access.